src/extract_bitmaps.c

```diff
@@ -1,5 +1,5 @@
 /*
-# Copyright (c) 2012, The Met Office, UK
+# Copyright (c) 2012 - 2015, The Met Office, UK
```

Comment:
If we haven't materially changed this file recently then we shouldn't claim it in the range of years. (If so, this applies to the other files too.)